gossip/src/duplicate_shred_handler.rs

```diff
@@ -283,6 +287,7 @@ mod tests {
             None::<fn(Slot) -> Option<Pubkey>>,
             timestamp(), // wallclock
             chunk_size,  // max_size
+            0,
```

Comment:
nit: This constant is duplicated with the 0 (4th argument) to `Shredder::new()`; it'd be nice to make a named variable at the top of the function and pass that down.

Arguably, `shred_version` should be an argument to `create_duplicate_proof()` as the callers of `create_duplicate_proof()` also have a `0` hard-coded into `DuplicateShredHandler::new()`